src/main/java/com/synopsys/integration/jira/common/model/response/IssueTypeResponseModel.java

```diff
@@ -91,4 +93,15 @@ public Integer getAvatarId() {
     public Map<String, JsonElement> getFields() {
         return fields;
     }
+
+    public Map<String, IssueCreatemetaFieldResponseModel> getTypedFields(Gson gson) {
+        return getFields().entrySet()
+                   .stream()
+                   .filter(entry -> entry.getKey() != "json")
```

Comment:
There should be a static name for this in IntRestComponent or somewhere in that hierarchy.